code/modules/admin/verbs/playsound.dm

```diff
@@ -70,6 +70,100 @@
 	SEND_SOUND(M, S)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Play Direct Mob Sound") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+///Takes an input from either proc/play_web_sound or the request manager and runs it through youtube-dl and prompts the user before playing it to the server.
+/proc/web_sound(mob/usr, input)
```

Comment:
```suggestion
/proc/web_sound(mob/user, input)
```

do this and this is ready to ship

Reply:
let me know if you really really need the `usr` meme for some reason and we can discuss that but `mob/usr` as a proc arg deeply disturbs me

Reply:
it honestly shouldn't compile.

usr, should really start getting treated like a reverse word in byond